src/USVPlugin.cpp

```diff
@@ -6,7 +6,8 @@ using namespace gazebo;
 using namespace gazebo_usv;
 
 USVPlugin::~USVPlugin() {
-    delete mWind;
+    if (mDirectForce) delete mDirectForce;
```

Comment:
delete nullptr; is valid C++. Do not check for NULL before deleting

Reply:
fixed